### ISSUE TITLE
ETSY003 does not detect use of wget in all situations

### ISSUE THREAD

**Issue by ziggythehamster** [etsy/foodcritic-rules#9]
Given this resource:

``` ruby
script "install_elasticsearch" do
  interpreter "bash"
  user "root"
  cwd "/tmp"
  code <<-EOH
    wget "#{node['elasticsearch']['download_url']}"
    tar xvzf #{node['elasticsearch']['filename']} -C #{node['elasticsearch']['dir']}
    ln -s "#{node['elasticsearch']['home_dir']}-#{node['elasticsearch']['version']}" #{node['elasticsearch']['home_dir']}
  EOH
  not_if "ls #{node['elasticsearch']['home_dir']}-#{node['elasticsearch']['version']}"
end
```

You would expect that ETSY003 would trip, but it does not.
